apps/shared/avifpng.c

```diff
@@ -196,6 +198,70 @@ static avifBool avifExtractExifAndXMP(png_structp png, png_infop info, avifBool
     return AVIF_TRUE;
 }
 
+// Converts a gamma-compressed value to linear.
+static double avifToLinear(double v, double gamma)
+{
+    return pow(v, gamma);
+}
+
+// Gamma-compresses a linear value using the sRGB transfer curve.
+// See section 8 of https://www.color.org/srgb.pdf
+#define SRGB_GAMMA_SLOPE 12.92
+#define SRGB_GAMMA_TRESHOLD 0.0031308
+static double avifToSrgbGamma(double linear)
+{
+    if (linear <= SRGB_GAMMA_TRESHOLD) {
+        return linear * SRGB_GAMMA_SLOPE;
+    } else {
+        return pow(linear, 1. / 2.4) * 1.055 - 0.055;
```

Comment:
Nit: 1. => 1.0

Make the same change to line 231.

See https://google.github.io/styleguide/cppguide.html#Floating_Literals

Reply:
Thanks, I didn't know this was in the style guide!